src/main/java/com/buyanywhere/productcatalog/controllers/v1/CategoryController.java

```diff
@@ -1 +1,25 @@
+package com.buyanywhere.productcatalog.controllers.v1;
 
+import com.buyanywhere.productcatalog.models.Category;
+import com.buyanywhere.productcatalog.repositories.CategoryRepository;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMethod;
```

Comment:
I don't see where this import is beeing used, please remove it and see if anythin breaks

Reply:
I see where it is used, thx